adoc/transfer.1.adoc

```diff
@@ -24,56 +24,86 @@ consistent, deterministic location.  The contents of a source directory will
+followed, and no other symbolic links are followed, transfered, or created on
+the destination endpoint. This behavior can be changed with the '--symlink' and
+'--recursive-symlinks' options.
 
 include::include/cli_autoactivate.adoc[]
 
 
 == Batch Input
 
 If you use SOURCE_PATH and DEST_PATH without the --batch flag, you
-will submit a single-file or single-directory transfer task. This has
-behavior similar to 'cp' and 'cp -r' across endpoints.
+will submit a single-item transfer task. This has behavior similar to
+'cp' across endpoints.
 
 Using '--batch', *globus transfer* can submit a task which transfers
-multiple files or directories. Paths to transfer are taken from stdin.
+multiple files directories or symlinks. Paths to transfer are taken from stdin.
 Lines are split on spaces, respecting quotes, and every line is treated as
-a file or directory to transfer.
+a separate item to transfer.
 
 Lines are of the form
-[--recursive] SOURCE_PATH DEST_PATH
-
+[--recursive] [--symlink] SOURCE_PATH DEST_PATH
 Empty lines and comments beginning with '#' are ignored.
 
-Batch only requires a SOURCE_ENDPOINT and a DEST_ENDPOINT before passing lines,
-on stdin, but if you pass SOURCE_PATH and/or DEST_PATH in the original command,
-these paths will be used as prefixes to their respective paths on stdin.
+'--recursive' and '--symlink' are mutually exclusive options that tell transfer
+to treat SOURCE_PATH as a dir to be transfered recursively, or a symlink that
+should be transfered as is rather than followed, respectively.
+Please note that these options cannot be used with the the command level
+'--recursive' and '--symlink' options despite sharing names and behavior, as
+the command level options are mutually exclusive with batch input.
+
+Batch only requires a SOURCE_ENDPOINT and a DEST_ENDPOINT with the original
+command before passing lines on stdin, but if you pass SOURCE_PATH and/or
+DEST_PATH in the original command, these paths will be used as prefixes to
+their respective paths on stdin.
 
 
 == OPTIONS
 
 *--batch*::
 
 Starts batch mode input for transfers, which takes lines in the form 
-[--recursive] SOURCE_PATH DEST_PATH from stdin to transfer multiple
-files and/or directories. See "Batch Input" above for more information.
+[--recursive] [--symlink] SOURCE_PATH DEST_PATH from stdin to transfer multiple
+files, directories, and/or symlinks. See "Batch Input" above for more
+information.
 
 *-r, --recursive*::
 
 If SOURCE_PATH and DEST_PATH are both directories, do a recursive transfer
 of all files and directories inside of the source directory to the destination
-directory. Note that for batch input, each directory must be marked as
-recursive individually unlike 'globus delete'.
+directory. For batch input each directory must use the '--recursive' option at
+the batch level, as this option is mutually exclusive with
+'--batch' at the command level. This option is also mutually
```

Comment:
I think you mean "mutually exclusive with `--recursive`"?

The growing potential for confusion here makes me wonder if we should split `transfer --batch` and `transfer srcep:src dstep:dst` into entirely separate commands.